src/workerd/api/actor.h

```diff
@@ -70,20 +101,20 @@ class DurableObjectId: public jsg::Object {
   friend class DurableObjectNamespace;
 };
 
-class DurableObject: public Fetcher {
+class DurableObject: public DurableObjectRpcStub {
```

Comment:
Why are `DurableObject` and `DurableObjectRpcStub` separate types? I think the new functionality should either be placed on `DurableObject` or on `Fetcher`, we don't need a new layer in between.

Reply:
Sorry I see this is answered in the comment above.